extensions/swagger-ui/deployment/src/main/java/io/quarkus/swaggerui/deployment/SwaggerUiProcessor.java

```diff
@@ -139,5 +313,236 @@ public void registerSwaggerUiServletExtension(SwaggerUiRecorder recorder,
          */
         @ConfigItem(defaultValue = "true")
         boolean enable;
+
+        /**
+         * The urls that will be included as options. By default the OpenAPI path will be used.
+         * Here you can override that and supply multiple urls that will appear in the TopBar plugin.
+         * Format is name=url,name=url (if name is omitted, url will be use as name)
+         */
+        @ConfigItem
+        Optional<String[]> urls;
+
+        /**
+         * If urls option is used, this will be the name of the default selection.
+         */
+        @ConfigItem(name = "urlsPrimaryName")
+        Optional<String> urlsPrimaryName;
+
+        /**
+         * The html title for the page.
+         */
+        @ConfigItem
+        Optional<String> title;
+
+        /**
+         * Swagger UI theme to be used.
+         */
+        @ConfigItem
+        Optional<ThemeHref> theme;
+
+        /**
+         * A footer for the html page. Nothing by default.
+         */
+        @ConfigItem
+        Optional<String> footer;
+
+        /**
+         * If set to true, enables deep linking for tags and operations.
+         */
+        @ConfigItem(name = "deepLinking")
+        Optional<Boolean> deepLinking;
+
+        /**
+         * Controls the display of operationId in operations list. The default is false.
+         */
+        @ConfigItem(name = "displayOperationId")
+        Optional<Boolean> displayOperationId;
+
+        /**
+         * The default expansion depth for models (set to -1 completely hide the models).
+         */
+        @ConfigItem(name = "defaultModelsExpandDepth")
+        Optional<Integer> defaultModelsExpandDepth;
+
+        /**
+         * The default expansion depth for the model on the model-example section.
+         */
+        @ConfigItem(name = "defaultModelExpandDepth")
+        Optional<Integer> defaultModelExpandDepth;
+
+        /**
+         * Controls how the model is shown when the API is first rendered.
+         */
+        @ConfigItem(name = "defaultModelRendering")
+        Optional<String> defaultModelRendering;
+
+        /**
+         * Controls the display of the request duration (in milliseconds) for "Try it out" requests.
+         */
+        @ConfigItem(name = "displayRequestDuration")
+        Optional<Boolean> displayRequestDuration;
```

Comment:
Make it a duration.

Reply:
It's just a flag that switch this on (in swagger-ui). So it's a boolean in swagger-ui, granted, badly named. see https://swagger.io/docs/open-source-tools/swagger-ui/usage/configuration/

Reply:
Ah, what tricked me was the `in milliseconds`.